go/vt/vtctl/reparentutil/replication.go

```diff
@@ -171,6 +171,8 @@ func StopReplicationAndBuildStatusMaps(
 		primaryStatusMap = map[string]*replicationdatapb.PrimaryStatus{}
 		m                sync.Mutex
 		errChan          = make(chan concurrency.Error)
+		tabletsReachable []*topodatapb.Tablet
+		allTablets       []*topodatapb.Tablet
 	)
 
 	groupCtx, groupCancel := context.WithTimeout(ctx, waitReplicasTimeout)
```

Comment:
What is the full timeout for ERS? what do we expect the typical value for `waitReplicasTimeout` to be?

Reply:
Both are configurable, if you call ERS from the vtctlclient, then the default action_timeout (total ERS time) is 1 hour and `waitReplicasTimeout` is also configurable which defaults to 30 seconds.
However, if we want ERS to be able to proceed with more than 1 failures, we need the action_timeout to be more than wait_replicas_timeout.